File: 프로그래머스/lv0/120841. 점의 위치 구하기/점의 위치 구하기.js
function solution(dot) {
    let x = dot[0];
    let y = dot[1];
    
    if(0 < x && 0 < y) {
        return 1;
    } else if(x < 0 && 0 < y) {
        return 2;
    } else if(x < 0 && y < 0) {
        return 3;
    } else if(0 < x && y < 0) {
        return 4;
    }
}

File: 프로그래머스/lv0/120841. 점의 위치 구하기/점의 위치 구하기.java
class Solution {
    public int solution(int[] dot) {
        int x = dot[0];
        int y = dot[1];
        return x >= 1 && y >= 1 ? 1 : x < 0 && y >= 1 ? 2 : x < 0 && y < 0 ? 3 : 4;
    }
}